src/components/screen/SearchUser.js

```diff
@@ -102,107 +103,101 @@ const StyledSearchImage = styled.Image`
 `;
 
 type Props = {
-
+  navigation: NavigationScreenProps,
 };
 type State = {
   users: Friend[],
   searchedUsers: Friend[],
 };
 
-class Screen extends Component<Props, State> {
-  static navigationOptions = {
-    title: 'Search User',
-  };
-
-  scrollY = new Animated.Value(0);
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      searchedUsers: userSampleData,
-      users: userSampleData,
-    };
-  }
+function Screen(props: Props) {
+  const profileModal = useContext(ProfileModalContext);
+  const [searchedUsers, setSearchedUsers] = useState(userSampleData);
+  const [users, setUsers] = useState(userSampleData);
+  const scrollY = new Animated.Value(0);
 
-  renderItem = (item: Friend, data: any) => {
+  const userListOnPress = (item: Friend) => {
+    if (profileModal && profileModal.dispatch) {
+      profileModal.dispatch({
+        type: 'show-modal',
+        payload: { user: item, deleteMode: true },
+      });
+    }
+  };
+  const renderItem = ({ item, index }: { item: Friend, index: number}) => {
     return (
       <UserListItem
-        id='user'
+        testID={`userListItem${index}`}
+        testObj={{ profileModal }}
         user={item}
-        onPress={() => this.showProfileModal(item, data)}
+        onPress={(e) => userListOnPress(item)}
```

Comment:
How to utilize `e`?

Reply:
I just unconsciously added `e` as a param. `e` is used nowhere... do you want me to remove this parameter `e`?

Reply:
I think it is better to remove it if not used.

Reply:
I removed it.